cmd/dep/testdata/harness_tests/README.md

```diff
@@ -19,29 +19,32 @@ The directory structure is as follows:
                 case2/
                 ...
 
-The test code itself simply walks down the directory tree, looking for `testcase.yaml` files.  These files can be as many levels down the tree as desired.  The test name will consist of the directory path from `testdata` to the test case directory itself.  In the above example, the test name would be `category1/subcategory1/case1`, and could be singled out with the `-run` option of `go test` (i.e. `go test github.com/golang/dep/cmp/dep -run Integration/category1/subcategory1/case1`).  New tests can be added simply by adding a new directory with the yaml file to the `testdata` tree.  There is no need for code modification - the new test will be included automatically.
-
-The yaml file needs to be accompanied by `initial` and `final` directories. The `initial` is copied verbatim into the test project before the `dep` commands are run, are the `manifest` and `lock` files in `final`, if present, are used to compare against the test project results after the commands.
-
-The `testcase.yaml` file has the following format:
-
-    # test case description
-
-    imports:
-        github.com/sdboyer/deptest: 0.8.0
-        github.com/sdboyer/deptestuno: a0196baa11ea047dd65037287451d36b861b00ea
-
-    initialVendors:
-        github.com/sdboyer/deptestdos: 1.0.1
-        github.com/sdboyer/deptesttres: a0196baa11ea047dd65037287451d36b861b00ea
-
-    commands:
-      - init
-      - ensure -update
-
-    finalVendors:
-        - github.com/sdboyer/deptest
-        - github.com/sdboyer/deptestdos
+The test code itself simply walks down the directory tree, looking for `testcase.json` files.  These files can be as many levels down the tree as desired.  The test name will consist of the directory path from `testdata` to the test case directory itself.  In the above example, the test name would be `category1/subcategory1/case1`, and could be singled out with the `-run` option of `go test` (i.e. `go test github.com/golang/dep/cmp/dep -run Integration/category1/subcategory1/case1`).  New tests can be added simply by adding a new directory with the json file to the `testdata` tree.  There is no need for code modification - the new test will be included automatically.
```

Comment:
While wrap may not matter overmuch, I think it'd be worth wrapping these at 80; this is a file that likely will be read in a terminal/editors.